README.md

```diff
@@ -2,16 +2,42 @@
 
 [![Streamlit App](https://static.streamlit.io/badges/streamlit_badge_black_white.svg)](https://blackary-mpa-config-streamlit-app-0bp2ol.streamlitapp.com/)
 
-An example of how to use a page_config.yaml file to control the ordering, icons, and nesting
-of the pages in the sidebar of a Multi-Page Streamlit App.
+<img width="600" src="https://user-images.githubusercontent.com/7164864/178483961-1f9abf49-eb44-455f-9362-37951daf055a.gif">
 
-Features include:
+### Summary 
 
-- Controling the order of the pages displayed
-- Nesting related apps within a "section"
-- Adding a created date on each app, and automatically adding a 🆕 icon
-- Automatically adding the icon and title at the top of each page
-- Displaying some pages in wide-mode
+Streamlit recently [released multi-page apps](https://blog.streamlit.io/introducing-multipage-apps/) 🎉 where page filenames are the source of truth for page settings. 
 
-Note that this depends on an API that may change, and is not designed to be used this way,
+In this repository, we show a prototype on how to use a [page_config.yaml](https://github.com/blackary/mpa-config/blob/main/page_config.yaml) to control the ordering, icons, and nesting of the pages in the sidebar of a multi-page Streamlit app.
+
+Main features include:
+
+- **Decoupling page settings from filenames.** No need to use indices or emojis in filenames to handle page titles, icons or ordering! You can name your Python scripts however you want.
+- **Controling the order of the pages.** Adding pages in the config in the order you want them to appear in the app.
+- **Nesting related pages within a "section".** Using `sections` in page_config.yaml will add un-clickable placeholders that group pages together.
+- **Automatically adding the icon and title at the top of each page.** Bringing consistency to your pages!
+
+⚠️ This project depends on an API that may change, and is not designed to be used this way,
 so this will probably NOT work long-term.
+
+
+### Get started
+
+1. Install requirements `pip install -r requirements.txt` 
+2. Run the Streamlit app `streamlit run streamlit_app.py`
+3. Play with the page_config.yaml!
+
+
+### Documentation
+
+Supported parameters in page_config.yaml for `pages`:
+- page_name: Page name.
+- icon: Emoji you want to use as an icon
+- created_date: App automatically adds a 🆕 flag to the page title if it is under 30 days old!
+- script_path: Path to your Streamlit script for that page. Defaults to `snake_case(page_name).py`
+- layout: Choose between 'wide' or 'centered'. Defaults to `'centered'`
+- deprecated: If true, the page will not be displayed in the app. Defaults to `'false'`
```

Comment:
Wondering if `visible` is a better keyword for this btw

Reply:
Visible is a good call